src/Domain/Sale/Application/CashierService.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * This file is part of the php-ddd project.
+ *
+ * (c) Yannick Voyer <star.yvoyer@gmail.com> (http://github.com/yvoyer)
+ */
+
+namespace Example\Domain\Sale\Application;
+
+use Example\Domain\Administration\DomainModel\Event\CandidateWasHired;
+use Example\Domain\Common\Application\EventPublisher;
+use Example\Domain\Common\DomainModel\JobTitle;
+use Example\Domain\Sale\DomainModel\Cashier;
+use Example\Domain\Sale\DomainModel\EmployeeRepository;
+use Example\Domain\Sale\DomainModel\Identity\EmployeeId;
+
+final class CashierService
```

Comment:
Depends on how we want to add Services in general @yvoyer. Will we keep Application Services that include several methods like in a CRUD pattern. Or will the entire Application work with a Message Bus for Commands and Handlers respecting CQRS.